src/subsonic/subsonicrequest.cpp

```diff
@@ -549,19 +559,14 @@ QString SubsonicRequest::ParseSong(Song &song, const QJsonObject &json_object, c
 
   QString cover_id;
   if (use_album_id_for_album_covers()) {
```

Comment:
```suggestion
  if (use_album_id_for_album_covers() && !album_cover_id.isEmpty()) {
```

Reply:
I tested that and it works. I committed it to the branch.
Thanks for the review!